eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -8,8 +8,23 @@ parameters:
   TargetDocRepoName: 'not-specified'
   Environment: nuget
 stages:
+  - stage: pre-release-validation
```

Comment:
We shouldn't need another stage for this check. It should be next to the verify changelog step below. 

We should also ensure this doesn't cause issue for folks that aren't generated or don't have the tsp-location.yaml or autorest.md as Jesse mentioned.

Reply:
It just displays a warning of missing those configuration files without other impacts. Does this matter?

Reply:
> It just displays a warning of missing those configuration files without other impacts. Does this matter?

Yes.  We should not emit warnings for expected and normal scenarios.  At best, it trains people not to pay attention to warnings, at worst it could be confusing to those unfamiliar with our processes and tooling, such as partners working in our repository.